src/string_bitflip.cc

```diff
@@ -0,0 +1,20 @@
+#include "string_bitflip.h"
+
+#include "node.h"
+#include "v8.h"
+
+#include <string.h>
```

Comment:
You don't need string.h here, you do need stdint.h (officially - you'll get it implicitly from node.h and v8.h but explicit > implicit.)